internal/cli/atlas/quickstart/quick_start.go

```diff
@@ -16,26 +16,39 @@ package quickstart
 import (
 	"errors"
 	"fmt"
+	"os"
+	"os/signal"
 	"os/user"
 	"regexp"
 	"strconv"
 	"strings"
+	"syscall"
 
 	"github.com/AlecAivazis/survey/v2"
 	"github.com/mongodb/mongocli/internal/cli"
 	"github.com/mongodb/mongocli/internal/config"
 	"github.com/mongodb/mongocli/internal/convert"
 	"github.com/mongodb/mongocli/internal/flag"
+	"github.com/mongodb/mongocli/internal/mongosh"
 	"github.com/mongodb/mongocli/internal/net"
 	"github.com/mongodb/mongocli/internal/randgen"
 	"github.com/mongodb/mongocli/internal/search"
 	"github.com/mongodb/mongocli/internal/store"
 	"github.com/mongodb/mongocli/internal/usage"
+	"github.com/pkg/browser"
 	"github.com/spf13/cobra"
 	atlas "go.mongodb.org/atlas/mongodbatlas"
 )
 
-const quickstartTemplate = "Now you can connect to your Atlas cluster with: mongo -u %s -p %s %s\n"
+const quickstartTemplate = `
+Now you can connect to your Atlas cluster with: mongosh -u %s -p %s %s
+
+`
+const quickstartTemplateCloseHandler = `
+You can connect to your Atlas cluster with the following user: 
+username:%s 
+password:%s
```

Comment:
[nit] a little spacing
```suggestion
username: %s 
password: %s
```